pandas/_libs/tslibs/period.pyx

```diff
@@ -1408,6 +1408,25 @@ cdef class _Period(object):
 
     @property
     def qyear(self):
+        """
+        Return a year of the given date.
+
```

Comment:
year of the given date not appropriate . I'd say "Get the year component of the Period."